### ISSUE TITLE
eval import error

### ISSUE THREAD

**Issue by RayOct18** [RayOct18/pyobfuscator#1]
```bash
Traceback (most recent call last):
  File "/home/rayxie/anaconda3/envs/test/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/rayxie/anaconda3/envs/test/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/rayxie/anaconda3/envs/test/lib/python3.8/site-packages/pobfuscatory/__main__.py", line 62, in <module>
    obfuscator.obfuscate()
  File "/home/rayxie/anaconda3/envs/test/lib/python3.8/site-packages/pobfuscatory/obfuscator.py", line 164, in obfuscate
    self._process(scan.run)
  File "/home/rayxie/anaconda3/envs/test/lib/python3.8/site-packages/pobfuscatory/obfuscator.py", line 193, in _process
    func(file_dir)
  File "/home/rayxie/anaconda3/envs/test/lib/python3.8/site-packages/pobfuscatory/obfuscator.py", line 246, in run
    subclass(self.root, self.keys)._execute(line, f)
  File "/home/rayxie/anaconda3/envs/test/lib/python3.8/site-packages/pobfuscatory/obfuscator.py", line 344, in _execute
    exec(line)
  File "<string>", line 1
    from src.predict.model import (
                                            ^
SyntaxError: unexpected EOF while parsing

```

**Comment by RayOct18** [comment]
fix [1579eb4](https://github.com/RayOct18/pobfuscatory/commit/1579eb4389e01d6f8c33cbf4f3f70f6542d5a3c6)

**Comment by RayOct18** [comment]
fix [8f52ccd](https://github.com/RayOct18/pobfuscatory/commit/8f52ccd2c5c59abcf5675835b88e02ddc311251e)
